### DISCUSSION TITLE
Solve the problem of Query-to-File installation

### DISCUSSION THREAD

**Discussion by erdengk** [polypheny/Polypheny-DB#377] [Show and tell]
If you also encounter the following problem, you can try to solve it according to this method.

[kext load failed: -603947007](https://github.com/polypheny/Polypheny-DB/issues/306#issuecomment-997369490)

> Task :Main. main()
kext load failed: -603947007
mount_ osxfuse: the file system is not available (255)
18:37:25.488 ERROR [JavaFX Application Thread]: Could not load fxml

You can view the schemes mentioned in this website.

[osxfuse/issues/741](https://github.com/osxfuse/osxfuse/issues/741)

After I reinstall macfuse and sshfs, I can run the Query-to-File normally.

**Comment by vogti** [comment]
Thank you, @erdengk, for sharing this solution!
